python/sdk/tests/test_detect.py

```diff
@@ -16,6 +16,8 @@
 class TestDetect:
 
     def setup_class(self):
+        if not pytest.expected_account['features']['detect']:
```

Comment:
will you move this to just the four tests that should be failing (schedule/unschedule ones)